perf/benchmark-against-version.py

```diff
@@ -0,0 +1,108 @@
+#! /usr/bin/env python3
+import logging
+import os
+import subprocess
+import sys
+import venv
+from pathlib import Path
+from typing import Optional
+
```

Comment:
^ It would be helpful to have a brief explanation of what this program is for at the beginning of the file.

Reply:
If the script is executable, then it shouldn't have a `.py` extension (because it's a command and it doesn't matter to the user in which language it's implemented).